<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>export_form</class>
 <widget class="QWidget" name="export_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Data</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0,0,0,1,0">
   <item>
    <widget class="QFrame" name="banner_frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background-color: rgb(45, 45, 45);
color: rgb(230, 230, 230);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="icon_label">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Export Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="asset_label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Project: Job_name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="help_btn">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Go to documentation.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(190, 190, 190);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rig Variant:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="variant_cmb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Current rig variant. Set it here to save data for a different vairant.</string>
     </property>
     <item>
      <property name="text">
       <string> Default</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="filter_btn">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
color: rgb(175, 175, 175);</string>
        </property>
        <property name="text">
         <string>Data Type Filters</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="filter_frame">
        <property name="styleSheet">
         <string notr="true">QFrame{background-color: rgb(60, 60, 60);;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="filter_layout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="all_chx">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>All Data Types</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="bsh_chk">
           <property name="whatsThis">
            <string>blendShape</string>
           </property>
           <property name="text">
            <string>BlendShapes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="cls_chx">
           <property name="whatsThis">
            <string>cluster</string>
           </property>
           <property name="text">
            <string>Clusters</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="cnct_chx">
           <property name="whatsThis">
            <string>connections</string>
           </property>
           <property name="text">
            <string>Connections</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="dmush_chx">
           <property name="whatsThis">
            <string>deltaMush</string>
           </property>
           <property name="text">
            <string>DeltaMush</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cons_chx">
           <property name="whatsThis">
            <string>constraints</string>
           </property>
           <property name="text">
            <string>Constraints</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="ffd_chx">
           <property name="whatsThis">
            <string>lattice</string>
           </property>
           <property name="text">
            <string>Lattices</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="nld_chx">
           <property name="whatsThis">
            <string>nonLinear</string>
           </property>
           <property name="text">
            <string>Non-Linear</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="pose_chx">
           <property name="whatsThis">
            <string>pose</string>
           </property>
           <property name="text">
            <string>Pose Interpolators</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="sclp_chx">
           <property name="whatsThis">
            <string>sculpt</string>
           </property>
           <property name="text">
            <string>Sculpt Deformers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="sdk_chx">
           <property name="whatsThis">
            <string>sdk</string>
           </property>
           <property name="text">
            <string>Set Driven Keys</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cms_chx">
           <property name="whatsThis">
            <string>cMuscleSystem</string>
           </property>
           <property name="text">
            <string>cMuscleSystem</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="skn_chx">
           <property name="whatsThis">
            <string>skinCluster</string>
           </property>
           <property name="text">
            <string>SkinClusters</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="attr_chx">
           <property name="whatsThis">
            <string>udAttributes</string>
           </property>
           <property name="text">
            <string>User Defined Attributes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nodes:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="node_list">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Nodes &amp; geometry to list data for export.</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="list_btn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Add selected nodes to list.</string>
           </property>
           <property name="text">
            <string>List Selected Nodes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="list_model_btn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>List selected nodes under model_GRP hierarchy.</string>
           </property>
           <property name="text">
            <string>List Model_GRP Hierarchy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Deformers &amp; Data:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="data_list">
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Deformers &amp; data to select for export.</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fix_names_btn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Rename selected data and deformers to &quot;GEO_NAME_&quot; + the deformer suffix.</string>
           </property>
           <property name="text">
            <string>Fix Selected Names</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fix_all_names_btn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Rename all data and deformers in list to &quot;GEO_NAME_&quot; + the deformer suffix.</string>
           </property>
           <property name="text">
            <string>Fix All Names In List</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="stack_btn">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Export selected defomrer stack to asset / data / variant folder. (Right-click for more options.)</string>
       </property>
       <property name="text">
        <string>Export Deformer Stack For Selected Nodes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="export_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Export selected data to asset / data / variant folder. (Right-click for more options.)</string>
       </property>
       <property name="text">
        <string>Export Selected Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="custom_btn">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Export all custom rigging to asset / data / variant folder. (Right-click for more options.)</string>
         </property>
         <property name="text">
          <string>Export Custom Rigging</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ncloth_btn">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Export all keyable attribute states and values to asset / data / variant folder. (Right-click for more options.)</string>
         </property>
         <property name="text">
          <string>Export nCloth</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="ctrls_btn">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Export all control shapes data to asset / data / variant folder. (Right-click for more options.)</string>
         </property>
         <property name="text">
          <string>Export Ctrl Shapes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shaders_btn">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Export all control shapes data to asset / data / variant folder. (Right-click for more options.)</string>
         </property>
         <property name="text">
          <string>Export Anim Shaders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sttrs_btn">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Export all keyable attribute states and values to asset / data / variant folder. (Right-click for more options.)</string>
         </property>
         <property name="text">
          <string>Export Keyable Attributes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>help_btn</tabstop>
  <tabstop>variant_cmb</tabstop>
  <tabstop>list_btn</tabstop>
  <tabstop>list_model_btn</tabstop>
  <tabstop>fix_names_btn</tabstop>
  <tabstop>fix_all_names_btn</tabstop>
  <tabstop>stack_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
